Readme.md

```diff
@@ -0,0 +1,390 @@
+# BillboardTopTen
+
+[BillboardTopTen][heroku] is a web application that allows music enthusiasts to explore Billboard's top ten charts throughout history. Inspired by [Billboard charts][Billboard], BillboardTopTen is built using Ruby on Rails on the backend, React.JS on the Front-end, and a PostgreSQL database.
+
+
+[heroku]: https://billboardtopten.herokuapp.com/
+[Billboard]: http://www.billboard.com/charts
+
+##BillboardTopTen Features
+
+* Dynamic visualization of Billboard's top ten charts for every week
+* Week's top song playing while charts are displayed
+* Smooth music fade in fade out transitions between different charts
+* Date Picker for user exploration of top ten charts through history
+* Genre Picker for user exploration of different music genres
+* Album Images associated with each artist on the chart
+
+
+## BillboardTopTen Walk-through
+
+### BillboardTopTen Web Scraping and Search APIs
+
+BillboardTopTen depends on web scraping HTML form [Billboard charts][Billboard] to retrieve all of the charts. A Python script using [billboard.py][billboardpy] API for accessing music charts was written to scrape ten songs for each chart. Each song in a chart contained the following fields: title, artist, weeks, rank, spotifyId, and spotifyLink.
+
+[billboardpy]: https://github.com/guoguo12/billboard-charts
+
+#### Sample Web Scraping Script Snippet
+
+```python
+import billboard
+from time import sleep
+
+f = open('edm.txt', 'w')
+
+i = 0
+chart = billboard.ChartData('dance-electronic-songs')
+while chart.previousDate:
+    print chart.date
+    if len(chart) < 10:
+        chart = billboard.ChartData('dance-electronic-songs', chart.previousDate)
+
+    f.write("*****")
+    f.write("\n")
+    f.write(chart.date)
+    f.write("\n")
+
+    for x in range(0, 10):
+        f.write(str(chart[x].title))
+        f.write("\n")
+
+        f.write(str(chart[x].artist))
+        f.write("\n")
+
+    ...
+```
+Once all of the charts were scraped, a second script was used to parse data into a JSON. This JSON contains a large hash map with a chart's date as a key, and a value as a list top ten tracks.
+
+From the new JSON, another script was used to loop through the hash map and build queries which where used to retrieve track samples from iTunes.
+
+The queries were built by first searching Spotify's API and retrieving clean formats for track artist and title. Once built, queries are sent through a HTTP GET to search iTunes API for track samples. These samples where saved into another JSON map with a week's date as key and sample URLs as values. This new map will later be used to play music for each week.
+
+A final script was used to retrieve URLs for album images associated with each track in the week. Album images may be more difficult to find then tracks. The script required queries to Spotify, iTunes, and Last.fm's APIs. Each album image found was save to a JSON file mapping track and artist names as keys to the album image URL.
```

Comment:
Remove the sentence `Album images may be more difficult to find then tracks.` and the sentence after it. And the last sentence too.